lib/use-checkout-pricing.js

```diff
@@ -55,13 +55,9 @@ export default function useCheckoutPricing (initialInputs, handleError = throwEr
 
     addSubscriptions(subscriptions, checkoutPricing)
       .then(() => {
-        if (adjustments.length) {
-          checkoutPricing = addAdjustments(adjustments, checkoutPricing).reprice();
-        };
+        checkoutPricing = addAdjustments(adjustments, checkoutPricing);
```

Comment:
These changes were made in response to a test correctly failing for an input that I didn't account for in the original code.